### PR TITLE
Add the changes that involves Different Methods, Overloading.

### PR DESCRIPTION
**Pull request by GalburG**
I have created Data generator Util Classes that allow me to create different clients data.
Additionally used overloading method , setters and getters.

### CODE REVIEW

File: src/main/java/com/galbur/methodstask/Customer.java [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+
+public class Customer {
+    private String name;
+    private int age;
+    private String email;
+
+    public Customer(String name, int age, String email) {
+        this.name = name;
+        this.email = email;
+        if (age <= 1 || age >= 127) {
```
Comment (AndreiSecuQA) [inline src/main/java/com/galbur/methodstask/Customer.java]: Age can be 1, I would set the < operator instead

File: src/main/java/com/galbur/methodstask/Customer.java [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+    public void setName(String newName) {
+        this.name = newName;
+    }
+
+    public int getAge() {
+        return age;
+    }
+
+    public void setAge(int newAge) {
+        if (newAge <= 1 || newAge >= 127) {
```
Comment (AndreiSecuQA) [inline src/main/java/com/galbur/methodstask/Customer.java]: Same as above

**Review by AndreiSecuQA — APPROVED** [review]

**Review by AnastasiaNeghina — APPROVED** [review]

File: src/main/java/com/galbur/methodstask/Customer.java (hunk 1 above)
Comment (GalburG) [reply src/main/java/com/galbur/methodstask/Customer.java]: did that from the start, then during watching the video applied <=,  but definitely worth changing it. thank you for your review

**Review by Nadiafol — COMMENTED** [review]
good!

**Review by Nadiafol — COMMENTED** [review]
good!

**Pull request merged**